### ISSUE TITLE
Chat participant list is unreachable via keyboard

### ISSUE THREAD

**Issue by Woebin** [JohnXLivingston/peertube-plugin-livechat#727]
**Describe the bug**
This is a violation of the [Web Content Accessibility Guidelines (WCAG)](https://www.w3.org/WAI/standards-guidelines/wcag/) [SC 2.1.1 (A) Keyboard](https://www.w3.org/WAI/WCAG22/Understanding/keyboard).

Not everyone can use a mouse, other pointing device, or touchscreen. To make things usable for everyone, it's important for interactive elements to be usable via keyboard alone.

The participants list that can be expanded in the chat widget can be opened via keyboard, but its contents aren't reachable at all. Tabbing through the page, we never reach these elements:
![Chat widget with participant list highlighted.](https://github.com/user-attachments/assets/21d2b5d1-f52e-49ee-a688-171080fb6223)

**To Reproduce**
1. Tab through the interface to the participants list button.
2. Open the participants list.
3. Try to tab into the list.
4. See that it's unreachable.

**Expected behavior**
Everything that's accessible by mouse should also be accessible by keyboard. Using standard semantic HTML elements instead of building custom ones out of `<div>` and `<span>` elements will usually achieve this.

**Server:**
 - Peertube version: 7.2.1
 - peertube-plugin-livechat version: 14.0.0

**Desktop:**
 - OS: MacOS 15.5
 - Browser: Google Chrome Version 137.0.7151.120 (Official Build) (arm64)

**Additional context**
This issue was recorded as part of an accessibility audit carried out by [Axess Lab](https://axesslab.com/), contracted by [HowlRound Theatre Commons](https://howlround.com/).
